requirements/base.txt

```diff
@@ -63,15 +63,15 @@ psutil==5.9.0
     # via edx-django-utils
 pycparser==2.21
     # via cffi
-pycryptodomex==3.13.0
+pycryptodomex==3.14.0
     # via pyjwkest
 pyjwkest==1.4.2
     # via edx-drf-extensions
 pyjwt[crypto]==2.3.0
     # via
     #   drf-jwt
     #   edx-drf-extensions
-pymongo==4.0.1
+pymongo==3.12.3
```

Comment:
The version is being downgraded because it is now constrained inside `edx-opaque-keys`. See https://github.com/openedx/opaque-keys/pull/206 for details.